config/kubernetes-client/OWNERS

```diff
@@ -2,6 +2,7 @@
 
 reviewers:
   - justaugustus
+  - sai-adarsh
```

Comment:
@Sai-Adarsh the reviwers are defined as per https://github.com/kubernetes/community/tree/master/github-management#new-membership-coordinator. :)

/close